test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -488,3 +488,20 @@ Describe "Help failure cases" -Tags Feature {
         { & $command foobar -ErrorAction Stop } | Should -Throw -ErrorId "HelpNotFound,Microsoft.PowerShell.Commands.GetHelpCommand"
     }
 }
+
+Describe 'help renders when using a PAGER with a space in the path' -Tags 'CI' {
+    BeforeAll {
+        $SavedEnvPager = $env:PAGER
+        $env:PAGER = "C:\Program Files\Git\usr\bin\less.exe"
```

Comment:
As steve said on this line:

>For testing, we don't need to use an actual pager. If the test is to validate piping data to something that has a space in the path, why not just use something like:
>
>Join-Path $TestDrive "folder with spaces\pager.ps1"
>
>Where pager.ps1 is simply takes input from pipeline (some test text) and outputs a hash you can verify?

Reply:
@TravisEz13 I have updated the test to use this idea. :)